tuya.cpp

```diff
@@ -513,6 +514,44 @@ void DeRestPluginPrivate::handleTuyaClusterIndication(const deCONZ::ApsDataIndic
                         }
                     }
                     break;
+                    
+                    // Dimmer level for mode 1
+                    case 0x0202:
+                    {
+                        if (R_GetProductId(lightNode) == QLatin1String("Tuya_DIMSWITCH Earda Dimmer") ||
+                            R_GetProductId(lightNode) == QLatin1String("Tuya_DIMSWITCH EDM-1ZAA-EU"))
+                        {
+                            quint8 bri = static_cast<quint8>(data * 254 / 1000); // 0 to 1000 value
```

Comment:
`const qint64 bri = data * 254 / 1000; // 0 to 1000 value`

Don't need to cast here since `ResourceItem::setValue(qint64 val)` takes an qint64 anyway.

Reply:
done